credoai/lens/lens.py

```diff
@@ -162,11 +161,6 @@ def run(self):
         """
         Run the main loop across all the pipeline steps.
         """
-        if len(self.pipeline) == 0:
```

Comment:
This is connected to the comment on `_validate()`, if we are still allowing the user to manually add steps 1 by 1, we can still get here and have empty pipeline, if that is the case the if statement can be use to collect the error, rather then generate the pipeline like it used to before.

Reply:
I removed this validation